src/js/core/widget/core/MainTab.js

```diff
@@ -0,0 +1,264 @@
+/*global ns, define */
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*jslint nomen: true, plusplus: true */
+/**
+ * #Main Tab
+ * Main Tab component is dedicated to control page change in app.
+ * The component is positioned always on bottom of the screen.
+ *
+ * @class ns.widget.core.MainTab
+ * @extends ns.widget.BaseWidget
+ * @author Tomasz Lukawski <t.lukawski@samsung.com>
+ */
+(function (document, ns) {
+	"use strict";
+	//>>excludeStart("tauBuildExclude", pragmas.tauBuildExclude);
+	define(
+		[
+			"../../engine",
+			"../../event",
+			"../../util/selectors",
+			"../BaseWidget",
+			"../core",
+			"./Marquee"
+		],
+		function () {
+			//>>excludeEnd("tauBuildExclude");
+			var BaseWidget = ns.widget.BaseWidget,
+				engine = ns.engine,
+				utilSelectors = ns.util.selectors,
+				FILE_REGEXP = /[^\/]+$/,
+				classes = {
+					/**
+					 * Standard widget
+					 * @style ui-main-tab
+					 * @member ns.widget.core.MainTab
+					 */
+					MAIN_TAB: "ui-main-tab",
+					/**
+					 * Set Main Tab widget as active (visible)
+					 * @style ui-main-tab-active
+					 * @member ns.widget.core.MainTab
+					 */
+					ACTIVE: "ui-active",
```

Comment:
ui-main-tab-active or ui-active?
in documentation you use ui-main-tab-active which I personally prefer